iphone/Classes/TiContactsGroup.m

```diff
@@ -276,14 +276,15 @@ -(void)add:(id)arg
 		CNSaveRequest *saveRequest = [person getSaveRequestForAddToGroup:group];
 		if (saveRequest == nil) {
 			DebugLog(@"[ERROR] Unable to add");
+
 			return;
 		}
 		if (![ourContactStore executeSaveRequest:saveRequest error:&error]) {
 			[self throwException:[NSString stringWithFormat:@"Unable to add member to group: %@",[TiUtils messageFromError:error]]
 					   subreason:nil
 						location:CODELOCATION];
 		};
-		RELEASE_TO_NIL(saveRequest)
```

Comment:
these changes to TiContactsGroup cannot be done. See
https://jira.appcelerator.org/browse/TIMOB-20010
It was done before and I had to revert it because this causes issues.
see this commit:
https://github.com/appcelerator/titanium_mobile/commit/a0d4a50d51f1afe85f92cf9e0d2ce8cca08fcf2f